org-roam.el

```diff
@@ -622,7 +622,8 @@ it as FILE-PATH."
                            (setq type "cite")
                            (org-ref-split-and-strip-string path))
                           ("fuzzy" (list path))
-                          (_ (if (file-remote-p path)
+                          (_ (if (or (file-remote-p path)
+                                     (s-starts-with-p "//" path))
```

Comment:
I think this is fine, but we should wrap it in a helper function ala `(org-roam--is-url-p path)` or add a comment, before we forget why we did this in the first place.